modules/administration_manual/pages/configuration/files/encryption/enable-encryption-app.adoc

```diff
@@ -0,0 +1,55 @@
+[[enable-the-encryption-app]]
+= How to Enable the Encryption App
+Matthew Setter <matthew@matthewsetter.com>
+:keywords: encryption, occ
+:description: This guide will show you how to enable the encryption app in ownCloud.
+:experimental:
+:page-partial:
+
+Before you can use encryption you must enable the encryption app. 
+You can do this either from the command-line or from the Web-UI.
+
+== Enable Encryption From the Command-Line
+// tag::enable-encryption-app-via-command-line[]
+To enable the encryption app, run the following command:
+
+[source,console]
+----
+sudo -u www-data php occ app:enable encryption
+----
+
+If the encryption app successfully enables, then you should see the
+following confirmation:
+
+....
+encryption enabled
+....
+
+[NOTE] 
+====
+This should never happen, but the encryption app may not be packaged with your ownCloud installation. 
+If so, you will see the following output when you attempt to enable it:
+
+....
+encryption not found
+....
+
+If that happens, then you need to install it manually. 
+You can do this by cloning the encryption app, using the following command:
+
+[source,console]
+----
+git clone https://github.com/owncloud/encryption.git apps/encryption
+----
+====
+// end::enable-encryption-app-via-command-line[]
+
+== Enable Encryption From the Web-UI
+
+To enable encryption from the Web-UI:
+
+. Go to menu:Settings[Admin > Apps] and click on kbd:[Show disabled apps]
+. When the disabled apps are rendered click kbd:[Enable] under _"Default encryption module"_.
+. After that go to menu:Settings[Admin > Encryption], and enable kbd:[Enable server-side encryption].
+. Then, under _"Default encryption module"_, select the desired encryption type, whether _"Master Key"_ (recommended) or _"User-key"_.
+. Now you must log out and then log back in to initialize your encryption keys.
```

Comment:
``Now you must log out and then log back in to initialize your encryption keys.`` -->
``Now you must log out and then log back in so that your encryption keys are automatically initialized.``

Else the question raises if there is something manually to do additionally by the admin

Reply:
I don't see a practical difference in the wording of both sentences.